spark/src/main/java/io/ddf/spark/content/SchemaHandler.java

```diff
@@ -192,6 +199,7 @@ public static Schema getSchemaFromDataFrame(DataFrame rdd) {
         return row != null;
       });
       dataFrame = manager.getHiveContext().createDataFrame(appliedRdd.rdd(), newSchema);
+      dataFrame.cache();
```

Comment:
There are two ways to force cache here:
1/ using our representation handler..
2/ signal spark to do cache.
I avoid to use first approach because it will traverse datasource 2 times. The first one is dataframe.count, and the second one is representationhandler.cache().
That why I choose to use #2.
If you all think it's not good, I will remove and fix to use representation handler.